### PR TITLE
Garlic Powder

### PR DESCRIPTION
**Pull request by Daved27hundred**
#### Summary
Content "Adds Garlic Powder"

#### Purpose of change

Garlic Powder is a common seasoning and I'm pretty certain every household has some. In fact I'm pretty sure that it's used more often then whole garlic at least where I live. More options for seasoning is nice, can't put it any other way.

#### Describe the solution

created garlic powder item, added the recipe, added it as a possible ingredient to several recipes, some that already used garlic cloves.

#### Describe alternatives you've considered

making garlic appear in pantries more often.

### CODE REVIEW

**Comment by onura46** [comment]
I recommend adding this to the `SUS_spice_collection` item group so that it dynamically populates in kitchens. There are item groups for restaurant kitchens which might be of interest, as well: 

https://github.com/CleverRaven/Cataclysm-DDA/blob/edb21b9806a9bbe53d50c4d459273b0a93daa548/data/json/itemgroups/SUS/domestic.json#L633

https://github.com/CleverRaven/Cataclysm-DDA/blob/edb21b9806a9bbe53d50c4d459273b0a93daa548/data/json/itemgroups/food_service.json#L1047

File: data/json/recipes/recipe_food.json [hunk 1]
```diff
@@ -4115,6 +4116,38 @@
...
   },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "garlic_powder",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_OTHER",
+    "skill_used": "cooking",
+    "difficulty": 2,
+    "time": "45 m",
```
Comment (phoeo) [inline data/json/recipes/recipe_food.json:4127]: Doesn't it take 3-8 hours to dry?

Comment (Daved27hundred) [reply data/json/recipes/recipe_food.json:4127]: does it? I could alter the recipe then. I'm not sure people will be happy that it takes so long so I'll make it between 1-2 hours.

Comment (Daved27hundred) [reply data/json/recipes/recipe_food.json:4127]: after looking it up, it seems that is for a massive amount of garlic. the 45 minutes is for a single clove. I'm sure if you're dehydrating several bulbs worth of garlic it'd take 3-8 hours ingame.

Comment (phoeo) [reply data/json/recipes/recipe_food.json:4127]: It just seems that in every recipe I saw there is a long time for passive dehydration. For example, here it says "4-6 hours in oven, or 2-3 days in open air"

https://www.alphafoodie.com/how-to-make-garlic-powder/

Comment (Daved27hundred) [reply data/json/recipes/recipe_food.json:4127]: huh. I assumed the player was just speed-dehydrating the garlic if that even exists. Similar to how you dehydrate fish or meat within 20 minutes ingame.

Comment (Daved27hundred) [reply data/json/recipes/recipe_food.json:4127]: I'm, going to resolve this conversation if that's okay with you, it seems like this isn't just the powder but dehydrated foods as a whole.

**Pull request merged**